src/core/client/ui/components/v2/TextField/TextField.css

```diff
@@ -41,18 +52,23 @@
 }
 
 .colorRegular {
-  color: var(--v2-palette-input-value);
-  background-color: var(--v2-palette-input-background);
-  border: 1px solid var(--v2-palette-input-border);
-  &:disabled {
-    border-color: var(--v2-palette-input-disabled-foreground);
+  border: 1px solid $input-border;
+  &:focus {
+    border: 1px solid $input-regular-focused-border;
+    box-shadow: inset 0px 0px 0px 1px $input-regular-focused-border;
+  }
+}
+
+.colorDark {
+  border: 1px solid $input-border;
```

Comment:
the teal focus border + blue adornment button looked weird, so I added a `dark` variant for the text field where the focus border is blue.